### PR TITLE
Audio player Feature/lmsdemo 115

### PR DESCRIPTION
**Pull request by victazzz**

### CODE REVIEW

File: src/components/players/Audio/Audio.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+```js
+import { GlobalThemeProvider } from "../../../theme/provider";
```
Comment (pieecia) [inline src/components/players/Audio/Audio.md:2]: redundant import

File: src/components/players/Audio/Audio.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+import ImageModal from "../../../styleguide/ImageModal";
+
+import ThemeTester from "../../../styleguide/ThemeTester";
+import Authbtn from "../../../utils/components/authbtn";
+
+<React.Fragment>
+  <ThemeTester>
+    <AudioPlayer
+      url="https://api-stage.escolalms.com//storage/course/7/topic/736/audio/siwCEwd14f7mPtY1rmfxcYovSTF73wKN1Sx3F7SL.mp4"
```
Comment (qunabu) [inline src/components/players/Audio/Audio.md:11]: replace URL with something that won't change in future

File: src/components/players/Audio/Audio.tsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
+import * as React from "react";
+import styled, { withTheme } from "styled-components";
+import { Text } from "../../atoms/Typography/Text";
+
+interface TrackProps {
```
Comment (qunabu) [inline src/components/players/Audio/Audio.tsx:5]: https://microsoft.github.io/PowerBI-JavaScript/interfaces/_node_modules_typedoc_node_modules_typescript_lib_lib_dom_d_.audiotrack.html

File: src/components/players/Audio/Audio.tsx [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+      <Text size="14" noMargin>
+        no source url provided
+      </Text>
+    );
+  }
+
+  return (
+    <StyledWrapper>
+      {url && ( // eslint-disable-next-line jsx-a11y/media-has-caption
+        <audio
```
Comment (qunabu) [inline src/components/players/Audio/Audio.tsx:35]: don't use browser audio use VideoPlayer component of something similar 

it should look like videoplayer with same capaliticies

**Review by qunabu — CHANGES_REQUESTED** [review]

**Comment by qunabu** [comment]
use VideoPlayer from components it is based on https://github.com/cookpete/react-player and does player video with tracks 

you can rename this components to AudioVideoPlayer

**Review by qunabu — APPROVED** [review]

**Pull request merged**